src/Service.Tests/dab-config.MsSql.json

```diff
@@ -3792,4 +3800,4 @@
       ]
     }
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
these changes were auto-generated by the configgenerator